File: part 3/Guessing_Game.c
#include<stdio.h>
#include<stdlib.h>
#include<time.h>

void main(void)
{
	srand(time(0));
	int guess=0, hotColdIndicator=3;
	int randomNumber =rand()%10;
	printf("This is a guessing game , if you can guess the right number between 0-9 with 4 or less attempts you win\n ");
	for (int counter=0;counter<4; counter ++)
{
    printf("\nWhat's your guess ?");
     scanf("%d",&guess);
     if (guess==randomNumber){
         printf("\nYou win!!");
         exit(0);
          }
      if (abs(guess-randomNumber)<hotColdIndicator ){
          printf("\nHotter");
          } else {
              printf("\nColder");
              }
              hotColdIndicator=abs(guess-randomNumber);
    }
    printf("\nOut of guesses.");
}

File: Ask_Mazen-Sakr_python-Programming/Guessing_game.py
#--------------------------------------------------------------------------- 

# file : Guessing game 

# Author : Mazen Sakr 

#-------------------------------------------------------------


#main code : 
print("This is a guessing game : \n")
secretWord = "Help!!"
guessCount = 0
guess =  ""
while guess !=secretWord and guessCount < 3 :
    guess = input("Please input your guess : ")
    guessCount = guessCount + 1
if guess != secretWord :
    print("Out of guesses, you lose")
else :
    print ("You Win!!")